### ISSUE TITLE
Store Tensorflow log directory path to MongoDB

### ISSUE THREAD

**Issue by chovanecm** [chovanecm/sacred#1]
Whenever an experiment that uses TensorFlow summary is run, store the path of the summary (summaries) to the database.

**Comment by chovanecm** [comment]
Idea: use decorator pattern on SummaryWriter and inject it into all SummaryWriter calls in the code like that:

``` python
original = tf.train.SummaryWriter.__init__
@functools.wraps(tf.train.SummaryWriter.__init__)
def newinit(self, *args, **kwargs):
    original(self, *args, **kwargs)
    print((args, kwargs)) #Here we would catch the logdir attribute
tf.train.SummaryWriter.__init__ = newinit
```

**Comment by Qwlouse** [comment]
Yes, something like that should work. Maybe there should be some `enable_tf_support()` call or the like? I don't really feel comfortable with monkeypatching tensorflow _implicitly_.

**Comment by chovanecm** [comment]
or an annotation... I agree with you, not by default...

**Comment by Qwlouse** [comment]
Haha, man you're _fast_! 

Sidenote: you've called your milestone Version 0.6m1, but I think it would be better to build it on top of 0.7 right away. This release should roll out soonish, and contains lots of changes. You can find it in the `release/0.7.0` branch.

**Comment by chovanecm** [comment]
I just accidentaly read the notification before a lecture has ended :-)
Ok, I wanted to ask you about that, so I'll use 0.7

**Comment by chovanecm** [comment]
@Qwlouse It seems to work (please see the [changes](https://github.com/IDSIA/sacred/compare/release/0.7.0...chovanecm:feature/tf-integration) if acceptable).

I'm just not sure if this `info["tensorflow"] = {"logdirs": ["/tmp/logdir1", "/tmp/logdir2"]}` is really the format we want to avoid problems caused by bad design decision in the future.
Maybe something like `info["tensorflow"] = {"logdirs": [{"logdir": "/tmp/training"}, {"logdir": /tmp/validation"}]}` because we also want to add the measured values (like loss) in the future.

But let's say that it works as a proof of concept.

And flake8 still fails on something, so I need to fix it :-)
sacred/tensorflow_hooks/**init**.py:1:1: F401 '.tensorflow_hooks.log_summary_writer' imported but unused
sacred/tensorflow_hooks/tensorflow_hooks.py:5:1: D204 1 blank line required after class docstring
